enterprise/troubleshooting.md

```diff
@@ -82,3 +82,12 @@ SSL certificate and may also already be present in your browser explaining why
 you can open Private Packagist pages in your browser without an error. Please
 see the section about `Invalid x509 keypair` for more information on how to add
 your intermediate certificates to your certificate file.
+
+#### Moving the application to a new IP/host or behind a HTTP proxy
```

Comment:
Since this is trouble shooting, the headline should probably instead describe the problem: the exact error message the replicated console shows when the healthcheck wait has timed out.